internal/policy/commit/commit.go

```diff
@@ -158,11 +160,11 @@ func (c Commit) firstWord() (string, error) {
 
 	if c.Conventional != nil {
 		groups = parseHeader(c.msg)
-		if len(groups) != 6 {
+		if len(groups) != 7 {
```

Comment:
this was a bug in `master` with conventional commits, as regexp got changed in some other PR

Reply:
looks like it got fixed in #204, but never merged (?)